### DISCUSSION TITLE
Avances DOCUTRACE

### DISCUSSION THREAD

**Discussion by AliciaBustamanteUNEA** [TheMoys/24-25-IdSw1-SDR#22] [General]
Hola, esta es mi cuenta!

**Comment by TheMoys** [comment]
@AliciaBustamanteUNEA Hola Alicia, confirmo de recibido y contesto mediante gitHub.

**Comment by AliciaBustamanteUNEA** [comment]
Buenos días, Diego:

Recibido, estoy contestando desde mi email. Me confirmas si también lo
recibís.

Saludos!

*Alicia Bustamante Alonso*
Gerente
***@***.*** <https://www.uneatlantico.es/>
PARQUE CIENTÍFICO Y TECNOLÓGICO DE CANTABRIA
C/ Isabel Torres 21 · 39011 Santander · España
*Teléfono +34 942 244 244 · uneatlantico.es <http://www.uneatlantico.es/>*

**Reply by TheMoys** [reply]
Buen día Alicia, si lo he recibido, te estoy contestando desde gitHub. 

Mantendremos comunicación de esta manera si no te incomoda.

**Reply by mmasias** [reply]
### 🥹🥹🥹🥹

**Comment by AliciaBustamanteUNEA** [comment]
Perfecto, Diego.


*Alicia Bustamante Alonso*
Gerente
***@***.*** <https://www.uneatlantico.es/>
PARQUE CIENTÍFICO Y TECNOLÓGICO DE CANTABRIA
C/ Isabel Torres 21 · 39011 Santander · España
*Teléfono +34 942 244 244 · uneatlantico.es <http://www.uneatlantico.es/>*

**Comment by TheMoys** [comment]
Buen día @AliciaBustamanteUNEA 

Te escribo para cuadrar una reunión esta semana para mostrarte los avances de la documentación del proyecto y que nos des tu retroalimentación sobre el rumbo por el que va encaminado. 

En principio Manuel había propuesto las horas de clase para realizar la reunión (Jueves entre las 12:00 y las 14:00 o Viernes entre las 10:00 y 12:00) que son franjas horarias en las que tenemos disponibilidad completa, pero si te viene mejor algún otro día y hora nos los dices y ya iremos comentando.

Que pases un buen día.
Saludos

**Comment by AliciaBustamanteUNEA** [comment]
Buenos días, Diego y resto del equipo:

¿El viernes a las 10 os vendría bien? A mí, sí. He reservado, a esa hora,
el aula 1.5, así que si me confirmas nos vemos el viernes allí.

Saludos,


*Alicia Bustamante Alonso*
Gerente
***@***.*** <https://www.uneatlantico.es/>
PARQUE CIENTÍFICO Y TECNOLÓGICO DE CANTABRIA
C/ Isabel Torres 21 · 39011 Santander · España
*Teléfono +34 942 244 244 · uneatlantico.es <http://www.uneatlantico.es/>*

**Reply by TheMoys** [reply]
Perfecto, nos vemos el viernes a las 10.

Saludos

**Comment by TheMoys** [comment]
Buen día @AliciaBustamanteUNEA,

Hemos realizado unos ajustes en los diagramas que te mostramos la semana pasada y nos gustaría reunirnos contigo la próxima semana para que los veas y podamos discutir más profundamente los detalles necesarios para poder seguir con los siguientes diagramas.

Al igual que la semana pasada, podriamos hacerlo en las horas en las que tengamos la clase de Manuel (Jueves de 12:00 a 14:00 o Viernes de 10:00 a 12:00), pero si te viene mejor otro día y hora nos lo dices y ya iremos coordinando cuando mejor te convenga.

Quedamos atentos a tu respuesta, 

Saludos y feliz día

**Comment by AliciaBustamanteUNEA** [comment]
Buenas noches, Diego y resto del equipo:

¿Nos vemos el jueves a las 12 en el aula 0.4?

Saludos,

Alicia Bustamante Alonso
Gerente
Universidad Europea del Atlántico

El vie, 29 nov 2024, 17:10, Diego Moys ***@***.***> escribió:

> Buen día @AliciaBustamanteUNEA <https://github.com/AliciaBustamanteUNEA>,
>
> Hemos realizado unos ajustes en los diagramas que te mostramos la semana
> pasada y nos gustaría reunirnos contigo la próxima semana para que los veas
> y podamos discutir más profundamente los detalles necesarios para poder
> seguir con los siguientes diagramas.
>
> Al igual que la semana pasada, podriamos hacerlo en las horas en las que
> tengamos la clase de Manuel (Jueves de 12:00 a 14:00 o Viernes de 10:00 a
> 12:00), pero si te viene mejor otro día y hora nos lo dices y ya iremos
> coordinando cuando mejor te convenga.
>
> Quedamos atentos a tu respuesta,
>
> Saludos y feliz día
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/TheMoys/24-25-IdSw1-SDR/discussions/22#discussioncomment-11417399>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BMVUWOESUDESU7CZP74ODMT2DCGYPAVCNFSM6AAAAABRGEOJJSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBRG4ZTSOI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
Alicia Bustamante Alonso
Gerente
Universidad Europea del Atlántico

**Reply by TheMoys** [reply]
Perfecto Alicia, nos vemos el jueves a la 12:00

Hasta entonces.

**Comment by TheMoys** [comment]
@AliciaBustamanteUNEA 
Te envío el documento que contiene las capturas de pantalla de las interfaces, cabe destacar que por la cantidad de módulos y sus variaciones acorde al tiempo, hemos tomado a bien reducirlo al minimo producto viable desde el punto de vista del administrador porque si no, habría que hacer el doble de interfaces para contemplar todos los casos posibles y alteraciones.
Esperamos tus comentarios y si tienes alguna otra consulta podríamos agendar una reunión el día de mañana a la hora de la clase o a la hora que te venga bien ya que el proyecto se presenta el día viernes 20 a las 10:00 am.

[Docutrace Interfaces.pdf](https://github.com/user-attachments/files/18182239/Docutrace.Interfaces.pdf)

**Comment by AliciaBustamanteUNEA** [comment]
Buenos días, Diego y resto del equipo:

Tras haber revisado anoche el documento, os felicito ya que no tengo nada
que indicaros y, además, me parece que es una interfaz bastante intuitiva
desde el lado UX.

Espero que hagáis una buena presentación mañana. Me gustaría que me
contaráis qué tal os fue.

Un saludo,


*Alicia Bustamante Alonso*
Gerente
***@***.*** <https://www.uneatlantico.es/>
PARQUE CIENTÍFICO Y TECNOLÓGICO DE CANTABRIA
C/ Isabel Torres 21 · 39011 Santander · España
*Teléfono +34 942 244 244 · uneatlantico.es <http://www.uneatlantico.es/>*

**Reply by TheMoys** [reply]
Buenos dias, Alicia:

¡Gracias por tus comentarios! Estaremos más que contentos de comentarte como nos va en la presentación, esperamos haber cumplido tus expectativas y si lo tomas a bien incluso podría desarrollarse este proyecto como TFG por parte de alguno de los integrantes del equipo.

Saludos y feliz dìa.

**Comment by AliciaBustamanteUNEA** [comment]
Claro, Diego, si alguno queréis seguir con el proyecto como TFG no tengo
problema en colaborar.

Mañana me contáis.

Saludos,

*Alicia Bustamante Alonso*
Gerente
***@***.*** <https://www.uneatlantico.es/>
PARQUE CIENTÍFICO Y TECNOLÓGICO DE CANTABRIA
C/ Isabel Torres 21 · 39011 Santander · España
*Teléfono +34 942 244 244 · uneatlantico.es <http://www.uneatlantico.es/>*

**Comment by TheMoys** [comment]
Hola, @AliciaBustamanteUNEA 

Adjunto el resumen de los puntos tratados durante la reunión de hoy sobre el proyecto.

**Reunión – 07/10/2025**

**Alcance acordado:**

- Envío de mensajes
- Trazabilidad de documentos enviados
- Firma digital de documentos
- Gestión de usuarios
- Gestión de grupos

**Casos de uso fuera del alcance:**

- Notificaciones
- Envío de correos
- Auditoría de acciones realizadas por un usuario

**Puntos adicionales:**

- Enfocar la presentación en la necesidad del proyecto y en los aspectos que lo diferencian de las soluciones existentes en el mercado.
- Definir un título claro y explicativo para el proyecto.
- Revisar la normativa de la UE sobre notificaciones y firmas digitales, para garantizar que todo esté en conformidad.

@diegojiimenez @mmasias

**Comment by mmasias** [comment]
Por la criticidad del proyecto, a valoración de @AliciaBustamanteUNEA sugeriría poner privado el repo. 

Como es un fork de repo mio, @TheMoys: me dices si da problema.

**Reply by TheMoys** [reply]
Efectivamente, no me deja cambiar la visibilidad debido a que es un fork de tu repo

<img width="801" height="141" alt="image" src="https://github.com/user-attachments/assets/9e65aada-a796-4ac6-b00e-90412369ca14" />

**Reply by mmasias** [reply]
Solicitar desvinculación del fork a GitHub Support:

- GitHub puede desvincular manualmente un fork de su upstream, permitiendo cambiar visibilidad. El repositorio se convierte en standalone.
- Proceso: Settings → Support → Contact GitHub Support, explicar necesidad de desvincular fork.
- Implicaciones: mismo resultado que duplicar manualmente, pero conserva el historial de issues/PRs si los hay. Tarda entre 24-48 horas.
